Naughtya/Projects/Features/CoreFeature/Sources/Presentation/Views/Search/SearchView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SearchView.swift
+//  CoreFeature
+//
+//  Created by byo on 2023/07/20.
+//  Copyright © 2023 Naughtya. All rights reserved.
+//
+
+import SwiftUI
+
+public struct SearchView: View {
+    private static let todoUseCase: TodoUseCase = MockTodoUseCase()
+
+    @StateObject private var viewModel = SearchViewModel()
+    private let textFieldHeight = CGFloat(40)
```

Comment:
```suggestion
    private let textFieldHeight: CGFloat = 40.0
```

init을 사용하지않고 타입을 명시해서 바로 값을 사용하는 것은 어떨까요 ?

Reply:
더 Swift스럽겠네용